src/librustc_mir/build/mod.rs

```diff
@@ -372,7 +372,9 @@ fn should_abort_on_panic<'a, 'gcx, 'tcx>(tcx: TyCtxt<'a, 'gcx, 'tcx>,
     // unwind anyway. Don't stop them.
     if tcx.has_attr(tcx.hir.local_def_id(fn_id), "unwind") { return false; }
 
-    return true;
+    // FIXME(rust-lang/rust#48251) -- Had to disable abort-on-panic
+    // for backwardsa compatibility reasons.
```

Comment:
typo, not a big deal :)

> backwardsa

Reply:
are you KIDDING? That is the biggest of deals.